README.md

```diff
@@ -4,7 +4,9 @@
 
 ## [WIP] dream2nix - A framework for automated nix packaging
 
-dream2nix is a framework for automatically converting packages from other build systems to nix.
+dream2nix is a framework for automatically converting packages from other build
+systems to nix.
```

Comment:
What is the reason for putting in new lines here and in many other places? I think the markdown renderer should decide where to wrap the lines. Otherwise we end up with weird line wrappings on smaller screens.

Reply:
Doesn't it already decide that (on GitHub / mdbook at least)? The reason I did that was so that it doesn't go over 80ish columns so it's easier to view and edit the raw markdown (especially if you don't have soft wrapping or don't like soft wrapping in your editor...). But if it is already a problem we can change it back (on some other docs as well).

Reply:
Sounds like something *you* have to set in your editor.

Reply:
Oh actually the .md renderer ignores single new lines, so it wouldn't lead to rendering issues. But anyways, I think it is better to not manually wrap in text files containing large text blocks. Otherwise you'd always have to adjust the line breaks after editing content. Let's change that back.

Reply:
Okay, I'll do that later (if you don't do it first). Looks like maybe I should look into soft wrapping for Helix xD